### ISSUE TITLE
Depth smooth loss

### ISSUE THREAD

**Issue by zjulabwjt** [HKUST-3DV/DIM-SLAM#4]
I am confused about the depth smooth loss.I found in your now code use the **fake_depth_loss**
`            fake_depth_loss = torch.nn.functional.smooth_l1_loss(
                depth, torch.ones_like(depth) * 1.5, beta=0.1, reduction="sum")`
what it means in the code.Besides,I can't find the smooth loss in the paper.
![image](https://github.com/HKUST-3DV/DIM-SLAM/assets/129039345/dee91a48-4a62-46ae-a49b-da05a10379d3)

**Comment by poptree** [comment]
Hi, I will add it to the final version if needed. During reimplementation, I found smooth term only has a minor effect. Thus I didn't include it in this version.

Best,
Heng

**Comment by poptree** [comment]
The fake depth loss provides an initialization on depth to prevent the optimization trap into a singular solution or local minima. It is only used in the first 150 iterations of the initialization. If you have another initialization method, feel free to replace the fake depth loss with it, and see the effectiveness.

**Comment by poptree** [comment]
Let me know if you still have any questions.

Best,
Heng

**Comment by zjulabwjt** [comment]
> Let me know if you still have any questions.
> 
> Best, Heng

Thanks for your reply!And when will you open source the fully code?

**Comment by poptree** [comment]
Maybe ~June, depending on my workload.

Best
